resource/api/swagger.yml

```diff
@@ -222,6 +222,30 @@ paths:
           description: Internal Server Error - The server has encountered a situation it does not know how to handle.
           schema:
             $ref: '#/definitions/Error'
+  /websiteCreator/state/{contractAddress}:
+    get:
+      operationId: websiteCreatorGetterStatus
+      parameters:
+        - in: path
+          name: contractAddress
+          type: string
+          required: true
```

Comment:
if you are using required: true, x-nullable:false should be set i think ?

Reply:
I tried and it changes nothing in generated files
I think that it is useless as it's a path parameter, so it has to be != null

Reply:
good point. But adding it will be consistent with all the other endpoints, so if it's not harmful i prefer to add it :-).